src/neptune/metadata_containers/run.py

```diff
@@ -451,6 +463,35 @@ def _write_initial_attributes(self):
             # upload default sources ONLY if creating a new run
             upload_source_code(source_files=self._source_files, run=self)
 
+    def _track_dependencies(self) -> None:
+        if self._dependencies == "infer":
+            # using pipreqs here
+            file_name = "neptune-tracked-dependencies.txt"
+            status_code = subprocess.check_call(["pipreqs", "--force", "--savepath", file_name, "."])
```

Comment:
How about using this package directly instead of console script? https://github.com/bndr/pipreqs/blob/master/pipreqs/pipreqs.py